### PR TITLE
chore(examples/values): default to unique builder

### PR DESCRIPTION
**Pull request by thbcmlowk**
## Description

Enabling the unique builder feature for `substra-ci`

## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

File: examples/values/backend-org-1.yaml [hunk 1]
```diff
@@ -125,6 +125,13 @@ orchestrator:
     - mychannel:
         restricted: false
         model_export_enabled: true
+        chaincode:
+          name: mycc
```
Comment (guilhem-barthes) [inline examples/values/backend-org-1.yaml:129]: We don't have a distributed mode, why to add these values?

Comment (thbcmlowk) [reply examples/values/backend-org-1.yaml:129]: Side-effect of the rebase (distributed mode was still a thing at the time I created those files). Removing it, thank you.

**Pull request closed without merging**